lib/utils/matrix_sdk_extensions/matrix_locals.dart

```diff
@@ -311,4 +296,12 @@ class MatrixLocals extends MatrixLocalizations {
 
   @override
   String get unknownUser => l10n.user;
+
+  @override
+  String setMeAsAdmin() => l10n.setMeAsAdmin;
```

Comment:
where is this used ? thought it was for toast

Reply:
Now, when some one set me as admin, the event "You now are the admin" in chat will be show